install.sh

```diff
@@ -17,11 +17,19 @@ EOF
   exit 2
 }
 
+INSTALL_DIRS="  ${BINDIR} \
+                ${XDG_BIN_HOME} \
+                ${HOME}/.local/bin \
+                ./bin"
+
 parse_args() {
-  BINDIR=${BINDIR:-./bin}
+  for dir in ${INSTALL_DIRS}; do
+    [ -d "${dir}" ] && { BINDIR="${dir}"; break; }
```

Comment:
an `if` is preferred in such cases
```suggestion
    if [ -d "$dir" ]; then
        BINDIR=$dir
        break
    fi
```